packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -968,6 +968,28 @@ Specifies a particular platform or device to match on
 }
 ```
 
+#### Device Condition object
+
+<ApiLifecycle access="ie" /><br>
+
```

Comment:
Please add the note here, as per out guidance:

```
> **Note:** This object is only available for Okta Identity Engine.
```

Reply:
Thanks will do